entropylab/results_backend/sqlalchemy/__init__.py

```diff
@@ -1,7 +1,14 @@
 from . import db
 from .db import SqlAlchemyDB
 
-from entropylab.results_backend.sqlalchemy.db_initializer import _DbInitializer
+from entropylab.results_backend.sqlalchemy.db_initializer import (
+    _DbInitializer,
+    _DbUpgrader,
+)
+
+
+def init_db(path: str):
+    SqlAlchemyDB(path)
```

Comment:
I think the init_db logic should be here, and the SqlAlchemyDB should call this function, not the other way around
It will be clearer

Reply:
Thanks. The `init_db()` logic resides mostly in the `_DbInitializer` class and is quite complex. I'd much rather leave it there because it's an entire class.

The method is just for readability. It might not be intuitive that this is what the SqlAlchemyDB ctor does so this method makes it more explicit in code.